canonical-web-and-design/netplan.io
# Eleventy

## Done

- Remove compress template
- Fix current page navigation highlight

## QA
- Check that pages render with correct styles
- Check that navigation shows which page you are on
